README.rst

```diff
@@ -1,13 +1,13 @@
 ruyaml
 ======
 
-``ruyaml`` package is a fork of ``ruamel.yaml`` aimed to made in order to
+``ruyaml`` package is a fork of ``ruyaml`` aimed to made in order to
```

Comment:
Uhh this is probably the one instance that shouldn't be changed

Reply:
Heh. Right.

Reply:
Yep. Right. I updated my "cleanup2" PR accordingly.